misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -24,8 +24,6 @@ data class WebConfig @JvmOverloads constructor(
    * checks may fail and k8s will kill the container, even though it might be perfectly healthy. This
    * can cause cascading failures by sending more requests to other containers, resulting in longer
    * queues and more health checks failures.
-   *
-   * TODO(rhall): make this required
```

Comment:
we use this internally everywhere now, so no more todo from someone who won't ever touch this code again